### PR TITLE
create new target and labels for mpi, sequential and merge mps tests

### PR DESCRIPTION
**Pull request by AbdoulbariZakir**

### CODE REVIEW

File: tests/CMakeLists.txt [hunk 1]
```diff
@@ -39,11 +39,23 @@ if (PYTHON_MODULE_pytest_FOUND AND PYTHON_MODULE_numpy_FOUND)
...
 	# benders end to end tests
     add_test(
-        NAME benders_end_to_end
-        COMMAND Python3::Interpreter -m pytest --installDir=$<TARGET_FILE_DIR:lp_namer> test_bendersEndToEnd.py
+        NAME sequential_benders_end_to_end
+        COMMAND Python3::Interpreter -m pytest --installDir=$<TARGET_FILE_DIR:lp_namer> test_bendersSequentialEndToEnd.py
         WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/end_to_end/benders
     )
-    set_property(TEST benders_end_to_end PROPERTY LABELS benders)
+    set_property(TEST sequential_benders_end_to_end PROPERTY LABELS sequential)
```
Comment (sgatto) [inline tests/CMakeLists.txt:46]:
```suggestion
    set_property(TEST sequential_benders_end_to_end PROPERTY LABELS "benders;benders-sequential")
```

File: tests/CMakeLists.txt [hunk 2]
```diff
@@ -39,11 +39,23 @@ if (PYTHON_MODULE_pytest_FOUND AND PYTHON_MODULE_numpy_FOUND)
...
+    add_test(
+        NAME mpibenders_benders_end_to_end
+        COMMAND Python3::Interpreter -m pytest --installDir=$<TARGET_FILE_DIR:lp_namer> test_bendersmpibendersEndToEnd.py
+        WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/end_to_end/benders
+    )
+    set_property(TEST mpibenders_benders_end_to_end PROPERTY LABELS mpibenders)
```
Comment (sgatto) [inline tests/CMakeLists.txt:52]:
```suggestion
    set_property(TEST mpibenders_benders_end_to_end PROPERTY LABELS "benders;benders-mpi")
```
this way will will run all benders tests by using the old label benders

**Review by sgatto — CHANGES_REQUESTED** [review]
add double labels in order to run oth benders tests using one label

File: tests/CMakeLists.txt (hunk 1 above)
Comment (AbdoulbariZakir) [reply tests/CMakeLists.txt:46]: done!

**Review by sgatto — APPROVED** [review]

**Pull request merged**